### ISSUE TITLE
replace webpack-isomorphic-tools with universal-webpack

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#2377]
Reduce duplication in our loader config by replacing `webpack-isomorphic-tools` tools with `universal-webpack`

See https://github.com/mozilla/addons-frontend/blob/2754e5c7faf719f65527fc853afb4243876d7df3/src/core/server/webpack-isomorphic-tools-config.js

See https://github.com/halt-hammerzeit/universal-webpack#universal-webpack-vs-webpack-isomorphic-tools

**Comment by kumar303** [comment]
Depends on upgrading to webpack 2 https://github.com/mozilla/web-ext/issues/787

**Comment by kumar303** [comment]
Hmm, I took a stab at just the dev server part and it feels a bit hairy. https://github.com/mozilla/addons-frontend/compare/master...kumar303:universal-iss2377

**Comment by kumar303** [comment]
Here are some alternatives that I'm going to look at:
* https://github.com/gajus/isomorphic-webpack
* https://github.com/Download/webpack-config-universal
* rolling our own with `target: node`

**Comment by kumar303** [comment]
Man, I really don't know about this. I think sticking with `webpack-isomorphic-tools` may be our best option right now.

I got kinda far on `isomorphic-webpack` but it's going to require us to split our configs even further and deal with a lot of differences between client/server. Maybe that path is still worth it but I felt fatigued by how hard it was. Their docs are very complex.

I checked out [Electrode](https://electrode.gitbooks.io/electrode/) a little bit. The specific part is their own [isomorphic-loader](https://github.com/electrode-io/isomorphic-loader). However, the explanation of how it works and the steps to install it are *nearly identical* to that of `webpack-isomorphic-tools`! How very suspicious. It does not seem to use it under the hood. I was also struck by how Electrode's own [config](https://github.com/electrode-io/electrode-confippet) library is also nearly identical to [node-config](https://github.com/lorenwest/node-config). Weird.
